crates/astria-conductor/src/conductor.rs

```diff
@@ -183,8 +224,9 @@ impl Conductor {
                 bearer_token: Some(cfg.celestia_bearer_token),
                 poll_interval: std::time::Duration::from_secs(3),
             };
-            // TODO ghi(https://github.com/astriaorg/astria/issues/470): add sync functionality to data availability reader
-            let reader = data_availability::Reader::new(
+            let da_reader = data_availability::Reader::new(
+                cfg.initial_da_block_height,
+                firm_commit_height,
```

Comment:
why are both of these needed? shouldn't only `firm_commit_height` be needed?

Reply:
These are both needed because the initial da block height is the celestia block height, and the firm height is the height from the sequencer. To be able to find the da block we need to start syncing from we need the firm commit when looking inside the da blocks, and the da block height to better find which da block to look in.